fixtures.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
We shouldn't forget to remove this when the pushing and pulling of projects and issues is done